test/kubemark/start-kubemark.sh

```diff
@@ -289,44 +315,73 @@ if [[ "${SCALEOUT_CLUSTER:-false}" == "true" ]]; then
     fi
   done
 
-  echo "DBG: Starting resource partition ..."
+  echo "DBG: Starting ${SCALEOUT_RP_COUNT} resource partitions ..."
   export KUBERNETES_TENANT_PARTITION=false
   export KUBERNETES_RESOURCE_PARTITION=true
-  create-kubemark-master
+  for (( rp_num=1; rp_num<=${SCALEOUT_RP_COUNT}; rp_num++ ))
+  do
+    export RESOURCE_PARTITION_SEQUENCE=${rp_num}
+    create-kubemark-master
+
+    RESOURCE_SERVER="http://"$(grep server "${RP_KUBECONFIG}-${rp_num}" | awk -F "/" '{print $3}')
```

Comment:
FYI. the API server ( RESOURCE_SERVER) is no longer needed since after PR 1001, the kubeconfig is used as media to the hollow-node-kubelet process